evalml/data_checks/datetime_format_data_check.py

```diff
@@ -39,11 +39,11 @@ def validate(self, X, y):
         Examples:
             >>> import pandas as pd
 
-            The column "dates" has a set of dates with hourly frequency appended to the end of a series of days, which is inconsistent
-            with the frequency of the previous 9 dates (1 day).
+            The column "dates" has a set of two dates with monthly frequency appended to the end of a set of two dates with hourly frequency that is also appended to the end
```

Comment:
nit, but it'd be clearer to say 
"The column 'dates' has a set of two dates with daily frequency, two dates with hourly frequency, and two dates with monthly frequency" or something of that sort